plugins/pagerduty/README.md

```diff
@@ -140,9 +156,9 @@ $ PAGERDUTY_TOKEN='<TOKEN>' yarn start
 
 This will proxy the request by adding an `Authorization` header with the provided token.
 
-### Optional configuration
+#### Optional configuration
 
-#### Annotating with Service ID
+##### Annotating with Service ID
```

Comment:
FIVE LEVELS DEEP? we have the technology

![image](https://github.com/backstage/backstage/assets/3097461/2bc588b7-48ea-46a5-8d3a-2e80dc970b6d)

Reply:
yeah didn't get too deep into re-writing this, just fixed the headings as there were about 5 first level headings :) i'll take a peek

Reply:
Oh just leave it as you had it, it's fine. You don't have to make it perfect as part of the cleanup :)